experimental/ner_spancat_compare/project.yml

```diff
@@ -0,0 +1,315 @@
+title: "Comparing SpanCat and NER using a corpus of biomedical literature (GENIA)"
```

Comment:
my review comments on the "readme" file should really have been made on the "yml" file!

Reply:
Ok! I regenerated the README as well!